content/pedersen-polynomial-commitment/en/pedersen-polynomial-commitment.md

```diff
@@ -78,6 +78,34 @@ In a sense, the prover is evaluating the polynomial using the coefficients to th
 
 The proof of correct evaluation is that the prover is able to separate the blinding terms from evaluation of the polynomial -- even though the prover does not know the discrete logs of $yG$ and $\pi B$.
 
+``` python
+import random
+from functools import reduce
+from py_ecc.bn128 import G1, add, eq, field_modulus, multiply
+
```

Comment:
We should include a warning above:

`# WARNING: Points are generated in this manner for convenience. In practice, the point's (x, y) value must be selected randomly and the discrete logs should never be known to anyone.`